File: application/service/Decorate/JsonPack/Resources.py

class Decorate():
    def packDomainToDict(self, resourceDomain):
        """
        :type resourceDomain: models.Resources.Domain.Resources_Domain
        """

        return {
            '_id': str(resourceDomain.getId()),
            'user': str(resourceDomain.getUser().getId()),
            'rubins': int(resourceDomain.getRubins()),
            'wood': int(resourceDomain.getWood()),
            'steel': int(resourceDomain.getSteel()),
            'stone': int(resourceDomain.getStone()),
            'eat': int(resourceDomain.getEat()),
            'gold': int(resourceDomain.getGold())
        }

    def getResources(self, userDomain):
        domain = super().getResources(userDomain)
        return self.packDomainToDict(domain)

File: public/js/factory/resources.js
define('factory/resources', [
    'model/resources'
], function (ResourcesDomain) {
    'use strict';

    var ResourcesFactory = AbstractFactory.extend({
        domain: ResourcesDomain,

        getDomainFromData: function (user, data) {
            var domain = new this.domain(user);
            domain.set(domain.RUBINS, data[domain.RUBINS]);
            domain.set(domain.WOOD, data[domain.WOOD]);
            domain.set(domain.STEEL, data[domain.STEEL]);
            domain.set(domain.STONE, data[domain.STONE]);
            domain.set(domain.EAT, data[domain.EAT]);
            domain.set(domain.GOLD, data[domain.GOLD]);

            return domain;
        }
    });
    return new ResourcesFactory();
});
